2_run/src/process_glm_output.R

```diff
@@ -19,11 +19,12 @@ combine_glm_output <- function(run_groups, outfile_template) {
   # combine into single feather file and write
   # truncating output for each time period to valid dates
   # (excluding burn-in and burn-out periods)
-  purrr::map2_df(run_groups$export_fl, run_groups$time_period, function(export_file, time_period) {
-    # Define time period begin and end dates
-    times <- strsplit(time_period,'_') %>% unlist()
-    begin <- sprintf('%s-01-01', times[1])
-    end <- sprintf('%s-12-31', times[2])
+  purrr::map2_df(run_groups$raw_meteo_fl, run_groups$export_fl, function(raw_meteo_fl, export_file) {
+    # Define time period begin and end dates from raw meteo_fl
+    meteo_data <- arrow::read_feather(raw_meteo_fl) %>%
+      mutate(time = as.Date(time))
```

Comment:
When I load these feather files, this column is already a `Date` class. Is that not the case for you?

Reply:
Circling back to this code since it is only being read in to get the begin and end date. It seems inefficient to read the entire file in when we are only doing this to get the range of the times. I would suggest adding `col_select = "time"` as an argument to `read_feather()` so that you aren't reading in unnecessary data columns.

Reply:
oooh I didn't know that was an option. Cool!